tensorflow/python/ops/clip_ops.py

```diff
@@ -212,6 +213,8 @@ def clip_by_norm(t, clip_norm, axes=None, name=None):
     values = ops.convert_to_tensor(
         t.values if isinstance(t, indexed_slices.IndexedSlices) else t,
         name="t")
+    clip_norm = tf.cast(math_ops.maximum(clip_norm, 0),dtype = values.dtype)
```

Comment:
You won't be able to use `tf.cast`.  I think it's in `math_ops`.

Reply:
Apologies.Yes, It should be `math_ops.cast`.